test/CMakeLists.txt

```diff
@@ -77,6 +77,14 @@ function(add_umf_test)
 
     set_tests_properties(${TEST_NAME} PROPERTIES LABELS "umf")
 
+    if(LINUX)
```

Comment:
is it needed for all the Linux tests now...? or just the one, new test...?

Reply:
Done.